WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt

```diff
@@ -240,18 +240,21 @@ class WPMainActivityViewModel @Inject constructor(
         val showFab = if (buildConfigWrapper.isJetpackApp) false else isOnMySitePageWithValidSite
         setMainFabUiState(showFab, site)
 
-        launch {
-            val currentVersionCode = buildConfigWrapper.getAppVersionCode()
-            val previousVersionCode = appPrefsWrapper.lastFeatureAnnouncementAppVersionCode
-
-            // only proceed to feature announcement logic if we are upgrading the app
-            if (previousVersionCode != 0 && previousVersionCode < currentVersionCode) {
-                if (canShowFeatureAnnouncement()) {
-                    analyticsTracker.track(Stat.FEATURE_ANNOUNCEMENT_SHOWN_ON_APP_UPGRADE)
-                    _onFeatureAnnouncementRequested.call()
+        // Do not proceed with feature announcement check for Jetpack
+        if (!buildConfigWrapper.isJetpackApp) {
```

Comment:
Suggestion (💡): Consider extracting this logic into a private method, which will be named accordingly and help remove the need for the extra comment in the first place, for example: `checkAndShowFeatureAnnouncementForWordPressApp()`

PS: We followed a similar approach for `removeFollowedBlogsPreferenceForJetpackApp()` within the `NotificationsSettingsFragment.java` class.

Reply:
Sure, np, I wrote it as is because it doesn't follow that pattern within the class. Easy enough to change. Thanks for the suggestion.

Reply:
In commit d0b152e